crates/anvil/core/src/types.rs

```diff
@@ -26,3 +28,12 @@ impl serde::Serialize for Work {
         }
     }
 }
+
+/// Represents the options used in `anvil_reorg`
+#[derive(Debug, Clone, Deserialize)]
+pub struct ReorgOptions {
+    // The depth of the reorg
+    pub depth: u64,
+    // List of transaction requests and blocks pairs to be mined into the new chain
+    pub tx_block_pairs: Vec<(TransactionRequest, u64)>,
```

Comment:
there's an issue with this because this disallows inclusion of raw transactions, I think we should support that feature?

Reply:
Good point. Added.